src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -498,7 +498,7 @@ def set_additional_ntp_source(self, ntp_source: List[str]):
             ntp_source_string = ntp_source
         else:
             raise TypeError(
-                f"ntp_source must be a string or a list of strings, got: {ntp_source}, type: {type(ntp_source)}"
+                f"ntp_source must be a string or a list of strings, got: {ntp_source}," f" type: {type(ntp_source)}"
```

Comment:
it may be missing a comma here? 
```suggestion
                f"ntp_source must be a string or a list of strings, got: {ntp_source},",  f" type: {type(ntp_source)}"
```

or remove the second f string

```suggestion
                f"ntp_source must be a string or a list of strings, got: {ntp_source}, type: {type(ntp_source)}"
```

Reply:
bad merge, I updated the commit